src/Components/GraduatesList/FilterButton.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Dropdown, Input } from "semantic-ui-react";
+function FilterButton(props) {
+  const filterOptions = [
+    {
+      key: "Node",
+      text: "Nodo",
+      value: "Node",
+      label: { color: "red", empty: true, circular: true }
+    },
+    {
+      key: "Module",
```

Comment:
I think key should be module-completed, not just module

Reply:
I agree, it should be more descriptive

Reply:
Thanks, I corrected that.